ddtrace/profiling/collector/_memalloc_tb.c

```diff
@@ -182,19 +229,17 @@ traceback_to_tuple(traceback_t* tb)
     PyObject* stack = PyTuple_New(tb->nframe);
 
     for (uint16_t nframe = 0; nframe < tb->nframe; nframe++) {
-        PyObject* frame_tuple = PyTuple_New(4);
-
         frame_t* frame = &tb->frames[nframe];
 
-        PyTuple_SET_ITEM(frame_tuple, 0, frame->filename);
-        Py_INCREF(frame->filename);
-        PyTuple_SET_ITEM(frame_tuple, 1, PyLong_FromUnsignedLong(frame->lineno));
-        PyTuple_SET_ITEM(frame_tuple, 2, frame->name);
-        Py_INCREF(frame->name);
-        /* Class name */
-        PyTuple_SET_ITEM(frame_tuple, 3, empty_string);
-        Py_INCREF(empty_string);
+        PyObject* frame_tuple = PyObject_CallFunctionObjArgs(ddframe_class,
```

Comment:
Would we be able to grab the definition of `DDFrame` from the event submodule and call that instead? This way we
- might save creating the namedtuple using the C API
- ensure that we use the same namedtuple class everywhere.